### ISSUE TITLE
pivpn install ends after configuration of static ip

### ISSUE THREAD

**Issue by Yezariael** [pivpn/pivpn#382]
# PiVPN Issue Template

## Console output of `curl install.pivpn.io | bash`
```
pi@raspberrypi:~ $  curl -L https://install.pivpn.io | bash
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   178  100   178    0     0    266      0 --:--:-- --:--:-- --:--:--   266
100 53505  100 53505    0     0  60604      0 --:--:-- --:--:-- --:--:-- 60604
:::
::: sudo will be used for the install.
::: Verifying free disk space...
:::
::: Checking apt-get for upgraded packages.... done!
:::
::: Your system is up to date! Continuing with PiVPN installation...
::: Static IP already configured.



```

## Console output of `pivpn`
```
pi@raspberrypi:~ $ pivpn
-bash: pivpn: command not found.


```

## Issue
I hope I can clarify my problem. You have to know that I'm almost a linux noob, sorry for that.

I tried to install pivpn several times on a raspberry pi 3 running raspbian jessie. I origninally tried the latest image "2017-07-05-raspbian-jessie" but as well "2017-06-21-raspbian-jessie".
With both installations I'm not able to install pivpn.
I start the installation and the IP configuration comes up. After I set a static IP the installer just exits with no error or something else as you can see above. I tried to configure a static IP before starting the setup but the same problem occurs.
I installed pivpn on a raspberry pi 2 with the latest raspbian jessie image of 2017-07-05 with no errors and the system is running fine but with the raspi 3 I'm not able to get it running.

If some information is missing please tell me what I have to do. 

Thanks in advance and regards
Yeza

## Console output of `cat /etc/os-release`
```
pi@raspberrypi:~ $ cat /etc/os-release
PRETTY_NAME="Raspbian GNU/Linux 8 (jessie)"
NAME="Raspbian GNU/Linux"
VERSION_ID="8"
VERSION="8 (jessie)"
ID=raspbian
ID_LIKE=debian
HOME_URL="http://www.raspbian.org/"
SUPPORT_URL="http://www.raspbian.org/RaspbianForums"
BUG_REPORT_URL="http://www.raspbian.org/RaspbianBugs"
```

## Console output of `uname -a`
```
pi@raspberrypi:~ $ uname -a
Linux raspberrypi 4.9.35-v7+ #1014 SMP Fri Jun 30 14:47:43 BST 2017 armv7l GNU/Linux
```

**Comment by levinus** [comment]
Same here, I'm also on a RB3 with Raspbian 8. The installation ended with the ip config and the pivpn command can not be found. Whats wrong here?

**Comment by cfcolaco** [comment]
@Yezariael @levinus 
Just a curiosity question ... if you are using raspbian fresh installs why are you using Jessie instead of stretch?

Will look into this and try to check if the problem is somewhere in the script ... but seems that might be something broken on your side ...

**Comment by goodmould** [comment]
Same issue here with pi zero running raspbian stretch.

pi@pizerovpn:~ $ curl -L https://install.pivpn.io | bash
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   178  100   178    0     0    174      0  0:00:01  0:00:01 --:--:--   174
100 53505  100 53505    0     0  29237      0  0:00:01  0:00:01 --:--:--  496k
:::
::: sudo will be used for the install.
::: Verifying free disk space...
:::
::: apt-get update has not been run today. Running now... done!
:::
::: Checking apt-get for upgraded packages.... done!
:::
::: Your system is up to date! Continuing with PiVPN installation...
pi@pizerovpn:~ $

pi@pizerovpn:~ $ cat /etc/os-release
PRETTY_NAME="Raspbian GNU/Linux 9 (stretch)"
NAME="Raspbian GNU/Linux"
VERSION_ID="9"
VERSION="9 (stretch)"
ID=raspbian
ID_LIKE=debian
HOME_URL="http://www.raspbian.org/"
SUPPORT_URL="http://www.raspbian.org/RaspbianForums"
BUG_REPORT_URL="http://www.raspbian.org/RaspbianBugs"

**Comment by Yezariael** [comment]
@cfcolaco as a linux noob I just did what the PiVPN page says.
> Strongly recommend using the latest Jessie Lite image but the normal Jessie image will work as well.

I tried Stretch as well but with it I get an error during generating the keys, I think a path changed.
If it would work with Stretch that would be fine for me as well :)

I did a couple of tries always with a completely fresh install of Jessie (and Stretch). No idea what should be the problem on my side.
Today i braught my Pi2 running Jessie, PiVPN and Pihole to work and I will try to copy an image to the problem-pi-3.

**Comment by levinus** [comment]
I actually used a fresh Stretch installation and everything worked fine, seems there was something wrong on my side before.

**Comment by cfcolaco** [comment]
@Yezariael Hmm Thank you for bringing that to my attention!!! we recently added the support for stretch .. so a few things need to be updated!!

@levinus Good to know!! However, I would like to be able to replicate this ... since I have seen this happening before but I don't remember why!! =/ and would be good to understand if there is something we can do on our side to help out!

**Comment by Yezariael** [comment]
@cfcolaco Today I did a clean install of Jessie and PiVPN on a completely other Raspi 3 and I have the same issue. After setting the IP to static mode the setup ends.

As a second I installed PiVPN on the same Raspi 3 with a clean install of Stretch. I get an error while trying to read the file in /etc/openvpn/easy-rsa/pki/index.txt.attr and have problems to access the VPN with Android.
I'll create a new issue for that.

**Comment by cfcolaco** [comment]
@Yezariael about the android there is already an issue regarding that ... the problem relies on the mobile apps! check here: #364 

I think is what you might be looking for!
About the rest. .. need to do more testing! eventually, tomorrow might go to hardware store and buy and extra RPI to have with me and do some testings ...

**Comment by borpin** [comment]
Trying to install from DietPi, same issue. Installer just quits.  Is there a log file anywhere?

**Comment by Yezariael** [comment]
@cfcolaco thanks for your great support!
Yes, I've already seen the thread #364 and tried the other app with success.

**Comment by cfcolaco** [comment]
@Yezariael so .. I still do not understand ... 

Can you exaplain me please exactly ... What have you done to fix??

**Comment by cfcolaco** [comment]
Closing this as per issue #388 .

Any questions read the mentioned please, if any issues persist please help us out and do things in a more organized way. 

Kind regards to you all.
